install.py

```diff
@@ -131,13 +131,10 @@ def download(url: str, filepath: str = os.getcwd()) -> None:
         pass
 except OSError:
     fancy("error", "Can't reach github. Check your internet connection")
-    exit(1)
+    sys.exit(1)
 
-if not Popen(
-    "find -H /var/lib/apt/lists -maxdepth 0 -mtime -7",
-    shell=True,
-    stdout=PIPE,
-).stdout:
+
+if not check_output(["find", "-H", "/var/lib/apt/lists", "-maxdepth", "0", "-mtime", "-7"]):
```

Comment:
Is this faster than using `glob` and `time`?

Reply:
I couldn't come up with a python alternative for this line... So I used subprocess. Can you suggest one?

Reply:
Or using `os.stat`

Reply:
Or using `os.stat()`

Reply:
Can you write the code snippet.

Reply:
```python
if [f for f in glob.glob('/var/lib/apt/lists/*') if os.stat(f).st_mtime >  604800]:
     print(1)
```

Reply:
Ouput: `1` means it's older than 1 week right?

Reply:
Yes, all files are older than 1 week in this case

Reply:
I just updated but it still outputs `1`...

Reply:
```python
import glob
import time

if not [f for f in glob.glob('/var/lib/apt/lists/*') if os.stat(f).st_mtime < time.time()  604800]:
     print(1)
```

Reply:
Your method ran faster
```
Summary
  'python3 g.py' ran
    1.15 ± 0.03 times faster than 'python3 f.py'
```

Implementing it